package.json

```diff
@@ -66,7 +66,7 @@
 			{
 				"id": "githubpr",
 				"name": "githubpr",
-				"fullName": "GitHub",
+				"fullName": "GitHub Pull Requests and Issues",
```

Comment:
The extension's name is actually "GitHub Pull Requests". What do you think of aligning with the extension name?

Reply:
It used to be "GitHub Pull Requests", then 5 years ago we changed it to "GitHub Pull Requests and Issues" because we added some issues stuff. Then, 1 year ago we changed it back to "GitHub Pull Requests" because the issues support was minimal.

Reply:
Hi ah yes then I will change it to GitHub Pull Requests

Reply:
Seems good. Though I do wonder if adding "and Issues" back into the name is in the not-so-distant future.